packages/evm/src/index.ts

```diff
@@ -1,4 +1,4 @@
-import { EOF } from './eof.js'
+//import { EOF } from './eof.js'
```

Comment:
Can we remove this?

Reply:
Yes I should remove this, however we should consider if we want to expose the `EOFContainer` :thinking: